tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -430,7 +444,47 @@ Open New Notebook
     Sleep    1
     Maybe Close Popup
 
+Clone Repo
+    [Documentation]    Clones git repo
+    [Arguments]    ${repo_url}
+    Navigate Home (Root folder) In JupyterLab Sidebar File Browser
```

Comment:
Missing tag as `Private Keyword`

Reply:
Done

Reply:
Sorry, it should be just `Private`

Reply:
Thinking again, as the code has been tested and it has 2 approvals this can be changed in another PR